tests/helpers.py

```diff
@@ -6,10 +6,11 @@
 from typing import Any, Dict, List, Optional
 from unittest import mock
 
+import pytest
```

Comment:
This ought to satisfy `mypy`.

```suggestion
import pytest  # type: ignore
```